std_msgs/Header header
object_recognition_v2/ObjectRecognition[] objects
